suites/pacific/rbd/tier-0_rbd.yaml

```diff
@@ -93,15 +93,15 @@ tests:
         install_packages:
           - ceph-common
           - rbd-nbd
-        node: node6
+        nodes: node6
```

Comment:
any reason why we changed from node-->nodes

Reply:
@ckulal Due to a recent change to tests/cephadm/test_client.py in pull/1666 package installation would be skipped now with old suites.
xxx.xx.92.24/job/rhceph-test-execution-pipeline/574/ - cephfs failure in this run seems to be for the same reason.

Reply:
Thanks for the clarification

Reply:
Is it possible to tweak test_client.py (to use both node or nodes), Instead of making changes to all the existing test suites?

Reply:
@ckulal sounds good.
as this PR is blocking RC build validation, shall we go ahead with current modifications and track tweaking of test_client.py in issue #1702 ?
(I might not be able to work on modifying test_client.py today)

Reply:
sure

Reply:
Made changes in the `test_client.py` to accept node/nodes.

Reply:
@pdhiran Thanks for the quick fix
Now I need to discard https://github.com/red-hat-storage/cephci/pull/1700/commits/8ebe7e4dfa78f7d346b7060f7369b70bb5e4a795
right ?

Reply:
Not a Necessity to discard. It would work with both node/nodes.

Reply:
Have dropped the respective PR as test_client is ow backward compatible, marking as resolved for now
please feel free to open if its not